<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);color: rgb(255, 255, 255);selection-background-color:rgb(140, 140, 140)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40,40,40);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0,0,0,0,0">
          <item>
           <widget class="QLineEdit" name="RecordingName">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>My Recording</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="ImageCountDisplay">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0 images</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="DurationDisplay">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Tahoma</family>
                <pointsize>12</pointsize>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>00:00:00.00</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="IsPlayingCheck">
            <property name="text">
             <string>_2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="IsRecordingCheck">
            <property name="text">
             <string>_1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="AudioCheck">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Audio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="TimeDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>115</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>115</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00:00.00</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="PlayBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>images/playBtn.png</normaloff>images/playBtn.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RecordBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>images/recordBtn.png</normaloff>images/recordBtn.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="NewBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Tahoma</family>
               </font>
              </property>
              <property name="text">
               <string>New</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="TimeSlider">
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>10</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(140, 0, 0);</string>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>TimeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSlider</sender>
   <signal>rangeChanged(int,int)</signal>
   <receiver>progressBar</receiver>
   <slot>setRange(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
